src/fe-common/core/chat-completion.c

```diff
@@ -1036,9 +1040,11 @@ static char *expand_escapes(const char *line, SERVER_REC *server,
 			/* newline .. we need to send another "send text"
 			   event to handle it (or actually the text before
 			   the newline..) */
-			if (ret != ptr) {
-				*ptr = '\0';
-				signal_emit("send text", 3, ret, server, item);
```

Comment:
in any case calling "send text" again to implement this "hack" would mean
"send text" "abc\ndef"
"send text" "abc"
being emitted, which seems wrong. the fe-text handler on the other hand catches the send text first and thus expand_escapes doesn't work at all (also sad ;-(